### PR TITLE
Added ability to get/set gravity on a physics character

### PR DESCRIPTION
**Pull request by louis-mclaughlin**

### CODE REVIEW

**Comment by seanpaultaylor** [comment]
Can you give some rationale for this ? Why not just update it on the PhysicsController for the whole scene? OR why not at least on PhysicsGraphicsObject ?

**Comment by louis-mclaughlin** [comment]
It allows you to have gravity pads/anti gravity zones in your game that are local to the player only without affecting the rest of the objects in the simulation. The most obvious example I can think of being https://www.youtube.com/watch?v=6zRL_B75ock . PhysicsRigidBody already supports this.

**Comment by seanpaultaylor** [comment]
It's fine for now. 
We can investigate later if we should add the method on PhysicsCollisionObject in case there are times that you want other local items that are not only the characters to receive this similar behavior too.

File: gameplay/src/PhysicsCharacter.h [hunk 1]
```diff
@@ -274,6 +298,7 @@ class PhysicsCharacter : public PhysicsGhostObject
     bool _physicsEnabled;
     float _mass;
     ActionInterface* _actionInterface;
+    gameplay::Vector3 _gravity;
```
Comment (seanpaultaylor) [inline gameplay/src/PhysicsCharacter.h]: Remove gameplay::

**Comment by seanpaultaylor** [comment]
It looks like from this change you are only applying gravity changes in jumping. This is not truly being simulated then.

**Comment by louis-mclaughlin** [comment]
Yeah this change only affects the vertical velocity, I didn't make gravity affect the horizontal velocity since the existing behaviour didn't. For example, without my change, using a gravity scene value of:

```gravity = -10, -9.8, 0``` 

doesn't have any effect on the characters movement along the x-axis.

I'm not sure how to implement the influence of gravity on the speed of movement, could you elaborate on this?

**Pull request closed without merging**